.git-blame-ignore-revs

```diff
@@ -10,3 +10,6 @@
 cb7a76d96c92aa9fc7b03f69148fb0c75dff191d
 5bbf61af8c8f12e6c05d07a696bf7d411b330a67
 d07546a6fef73fa4e1fb1c2f01535843d1fcc212
+
+# UTF-8 BOM changes
+12cdc5cd79321d71cc228d41775e03c8cc360fbe

```

Comment:
Wait a second... Shouldn't this be https://github.com/microsoft/terminal/commit/ddae2a1d49d604487d3c963e5eacbeb73861d986?

Reply:
how the heck did .. yes